index.html

```diff
@@ -142,7 +144,7 @@ <h2 id="footer-head" class="footer-heading">Contact me</h2>
                 </li>
                 <button type="submit" id="submit"  class="submitform">Get in touch</button>
                </ul>
-            </form>
+            </form> -->
```

Comment:
- [ ] Good Job for implementing the Project requirement But You commented out the `Form` section So the Form is not displaying as shown the screenshot  Kindly Fix the issues.
![contact](https://github.com/denigit5/Portfolio/assets/93598189/c287aec5-5bb0-401d-b4d0-a3bae243db4e)